experimental/oak_baremetal_app_crosvm/README.md

```diff
@@ -0,0 +1,5 @@
+# Oak Baremetal crosvm
+
+This version of the baremetal binary that can be run under
```

Comment:
This sentence does not have a verb, or am I missing it? Perhaps change to "This is a version of ..."

Reply:
Done.